src/Pages/Login.jsx

```diff
@@ -1,8 +1,51 @@
+import { Header, Footer, Aside } from "../Components/index";
```

Comment:
can use default imports

Reply:
Thank you for the suggestion. I'll look into it.